tools/dissector/packet-opendds.cpp

```diff
@@ -430,15 +439,24 @@ namespace OpenDDS
                    i < sample.content_filter_entries_.length();
                    i++)
                 {
+                  // Get Entry Value
                   const GUID_t &filter = sample.content_filter_entries_[i];
                   DCPS::GuidConverter converter(filter);
                   std::stringstream strm;
-                  strm << "filter [" << i << "] = " << converter << std::ends;
+                  std::string guid = strm.str();
```

Comment:
This doesn't look right -- `guid` will always be an empty string and the stringstream is unsued

Reply:
I discovered that this morning when I ran the content filter test. I have fixed it in the commits I'm about to push.